### PR TITLE
Feature: environments

### PR DESCRIPTION
**Pull request by vadymmarkov**
I know that it doesn't build on Linux atm, but I think this branch could be merged, so I could continue on reducing the amount of dependencies to fix it. We have sort of stable version https://github.com/hyperoslo/Flamingo/releases/tag/0.1.0 and `master` will be under continuous development.

Here we have:
- String-based environments: `Environment("production")`
- Working with system environment variables: 

``` swift
Env.add(value: "test", key: "key")
Env.value("key") // => test
Env.contains("key") // => true
Env.remove("key")
Env.contains("key") // => false
```

### CODE REVIEW

File: Sources/Flamingo/Utilities/Env.swift [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    return string
+  }
+
+  /**
+    Sets a new value for a given environment variable.
+
+    - Parameter key: The key.
+    - Parameter value: The value.
+  */
+  public static func add(value: String, key: String) {
```
Comment (onmyway133) [inline Sources/Flamingo/Utilities/Env.swift:33]: Can you use some kind of subscript? It will look familiar

Comment (vadymmarkov) [inline Sources/Flamingo/Utilities/Env.swift:33]: @onmyway133 You can't have static subscript.

Comment (onmyway133) [inline Sources/Flamingo/Utilities/Env.swift:33]: @vadymmarkov a global var could make it happen? This is just syntactic sugar 😄

Comment (RamonGilabert) [inline Sources/Flamingo/Utilities/Env.swift:33]: What is wrong with a static var, @onmyway133?

Comment (onmyway133) [inline Sources/Flamingo/Utilities/Env.swift:33]: @RamonGilabert nothing. I see that like `ENV["key"] = "value"` looks more familiar

Comment (vadymmarkov) [inline Sources/Flamingo/Utilities/Env.swift:33]: @onmyway133 I think we can do it, but I kinda afraid of using global vars tbh.

**Comment by vadymmarkov** [comment]
Can we merge this and @onmyway133 could create an issue to improve it with subscripts?

**Comment by onmyway133** [comment]
😁

**Pull request merged**